pkg/utils/json.go

```diff
@@ -73,3 +73,23 @@ func AreEqualJSON(s1, s2 string) (bool, error) {
 
 	return reflect.DeepEqual(o1, o2), nil
 }
+
+// AreEqualJSONBytes validate if two json byte arrays are equal
+func AreEqualJSONBytes(b1, b2 []byte) (bool, error) {
```

Comment:
Is it possible to use `AreEqualJSON` already present in the file?
If not, I think we can refactor `AreEqualJSON` func.